.circleci/config.yml

```diff
@@ -80,6 +80,9 @@ jobs:
       - run:
           name: Publish package
           command: cd dist && npm publish
+      - run:
+          name: Validate publish
+          command: cd ~/grommet-ci/templates/grommet-v2-template && yarn install && yarn build
```

Comment:
Probably should at least validate the release before publishing? Right now if something goes wrong here the release is happening anyway.
I wonder whether this needs to be in the build stage.

Reply:
moving the command to the build won't test the publishing itself. 
I agree we should consider having an additional flow for testing before publishing, but I'm currently not sure where and I don't think it should be done instead.

This validation tets could have saved me a lot of time on our recent release, and as long as it brings value we shouldn't wait with it, and leverage it as a baseline for future validations.

Reply:
What was the issue with the last release?

Reply:
incorrect publishing that could have been caught and raise a red flag with this validation. check out the recent commits on this file.